external-crates/move/crates/move-vm-config/src/runtime.rs

```diff
@@ -3,9 +3,13 @@
 
 use crate::verifier::{VerifierConfig, DEFAULT_MAX_CONSTANT_VECTOR_LEN};
 use move_binary_format::file_format_common::VERSION_MAX;
+use once_cell::sync::Lazy;
+use std::path::PathBuf;
 
 pub const DEFAULT_MAX_VALUE_NEST_DEPTH: u64 = 128;
 
+pub static DEFAULT_PROFILE_OUTPUT_PATH: Lazy<PathBuf> = Lazy::new(|| std::path::PathBuf::from("."));
```

Comment:
This seems unnecessary, could you explain why this line was needed, vs just calling `std::path::PathBuf::from(".")` inline?

Reply:
this variable is used by multiple locations to indicate the profiler is enabled, but there was no user-specified output path, seemed cleaner to define it once but not necessary

Reply:
I would say it's not really worth it, especially given `std::env::current_dir()` exists.